examples/jsm/nodes/core/AssignNode.js

```diff
@@ -0,0 +1,79 @@
+import TempNode from './TempNode.js';
+
+class AssignNode extends TempNode {
+
+	constructor( aNode, bNode ) {
+
+		super();
+
+		this.aNode = aNode;
+		this.bNode = bNode;
+
+	}
+
+	hasDependencies( builder ) {
+
+		return false;
+
+	}
+
+	getNodeType( builder, output ) {
+
+		const aNode = this.aNode;
+		const bNode = this.bNode;
+
+		const typeA = aNode.getNodeType( builder );
+		const typeB = bNode.getNodeType( builder );
+
+		if ( typeA === 'void' || typeB === 'void' ) {
+
+			return 'void';
+
+		} else {
+
+			return typeA;
+
+		}
+
+	}
+
+	generate( builder, output ) {
+
+		const aNode = this.aNode;
+		const bNode = this.bNode;
+
+		if ( aNode.isBufferNode === true || ( aNode.node && aNode.node.isBufferNode === true ) ) { // TODO: Maybe this can be moved to .construct()?
+
+			const nodeData = builder.getDataFromNode( aNode.isBufferNode ? aNode : aNode.node, builder.getShaderStage() );
+			const buffer = nodeData.uniformBuffer;
+			if ( buffer !== undefined ) {
+
+				builder.outComputeBuffer = buffer;
```

Comment:
I think we should be able to support multiples buffers in WebGL too?

Reply:
There could be only one output buffer (implemented as a render target). There can be multiple source buffers simultaneously.

Reply:
Maybe we can use `WebGLMultipleRenderTargets`... But this will require considerably changing WebGLNodeBuilder, I think (because currently it only allows writing to `diffuseColor`).

Reply:
> But this will require considerably changing WebGLNodeBuilder, I think (because currently it only allows writing to diffuseColor).

Or maybe just change `WebGLBuffer.setElement()` from just returning the value to assigning it to a texture. This, I think, can also solve the problem with `return stack`s...